templates/index.html

```diff
@@ -22,38 +22,61 @@
+                  <td><a href="https://maas.io" class="p-link--external">MAAS</a></td>
                   <td>
-                      <p>Documentation for the next generation OS from Ubuntu for all your IoT devices.</p>
-                      <p><a href="/core/en">Read the Ubuntu Core documentation&nbsp;&rsaquo;</a></p>
+                      <p>Documentation for MAAS (Metal as a Service).</p>
+                      <p><a href="/maas/2.1/en/">Read the MAAS documentation&nbsp;&rsaquo;</a></p>
                   </td>
               </tr>
               <tr>
-                  <td><a href="https://snapcraft.io" class="p-link--external">Snapcraft</a></td>
+                  <td><a href="https://landscape.canonical.com" class="p-link--external">Landscape</a></td>
                   <td>
-                      <p>Developer documentation to learn how to snap your apps for easy cross platform distribution and updates that just work.</p>
-                      <p><a href="https://snapcraft.io/docs" class="p-link--external">Read the Snapcraft documentation</a></p>
+                      <p>Documentation for Landscape.</p>
+                      <p><a href="/landscape/en">Read the Landscape documentation&nbsp;&rsaquo;</a></p>
                   </td>
               </tr>
               <tr>
                   <td><a href="https://conjure-up.io" class="p-link--external">Conjure-up</a></td>
                   <td>
                       <p>Documentation for Conjure-up.</p>
-                      <p><a href="/conjure-up/en">Read the Conjure-up documentation</a></p>
+                      <p><a href="/conjure-up/en">Read the Conjure-up documentation&nbsp;&rsaquo;</a></p>
                   </td>
               </tr>
               <tr>
                   <td><a href="https://github.com/canonicalltd/documentation-builder" class="p-link--external">documentation-builder</a></td>
                   <td>
                       <p>A tool for generating documentation HTML from Markdown in the standard Canonical format.</p>
-                      <p><a href="/documentation-builder">Read the documentation-builder documentation</a></p>
+                      <p><a href="/documentation-builder">Read the documentation-builder documentation&nbsp;&rsaquo;</a></p>
+                  </td>
+              </tr>
+          </tbody>
+      </table>
+
+      <h2>Externally hosted documentation</h2>
+      <table>
+          <thead>
+              <tr><th>Project</th><th>Description</th></tr>
+          </thead>
+          <tbody>
+              <tr>
+                  <td><a href="https://www.ubuntu.com/desktop" class="p-link--external">Official Ubuntu documentation</a></td>
+                  <td>
+                      <p>Documentation for the Ubuntu operating system.</p>
+                      <p><a href="https://XXXX" class="p-link--external">Read the official Ubuntu documentation</a></p>
+                  </td>
```

Comment:
It is "ready", it was mean to send you to https://help.ubuntu.com/, but I forgot the URL at the time. I've added it now.